### PR TITLE
Initial version.

### PR DESCRIPTION
**Pull request by ekoi**

### CODE REVIEW

File: src/StarRating.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const tooltipArray = [
+  "Terrible",
+  "Bad",
+  "Average",
+  "Great",
+  "Awesome",
+];
+interface RatingProps {
+  user_id: string
+  page_id: string
```
Comment (kerim1) [inline src/StarRating.tsx:14]: These properties are not used in the component. So guessing they can be removed?

File: src/StarRating.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  rate: number
+  readonly : boolean
+  handleRating: any
+}
+export default function StarRating({ user_id, page_id, rate , readonly, handleRating}: RatingProps) {
+  return (
+    <div>
```
Comment (kerim1) [inline src/StarRating.tsx]: Is the wrapper `<div>` required?

File: src/review.tsx [hunk 3]
```diff
@@ -1,7 +1,64 @@
...
+        </Form.Root>
     );
 }
+
+export default ReviewForm;
+
+
+ function handleSubmit(formData: FormData) {
+    console.log(formData);
+  }
```
Comment (kerim1) [inline src/review.tsx]: Function is not used, so can be removed.

File: src/review.tsx [hunk 4]
```diff
@@ -1,7 +1,64 @@
-import React from 'react';
+import React, {useState} from 'react';
+import * as Form from '@radix-ui/react-form';
+import StarRating from "./StarRating";
+
+interface FormParam{
+    id: string
+    user: string
+    handleRating: any
```
Comment (kerim1) [inline src/review.tsx:8]: The type for `handleRating` is not defined (is `any` now). Should be something like `(stars: number, reviewText: string) => void`.

File: src/review.tsx [hunk 5]
```diff
@@ -1,7 +1,64 @@
...
+    const [invalidStars, setInvalidStars] = useState(false);
+    const [success, setSuccess] = useState(false)
 
-export default function Review() {
     return (
-        <div>Review</div>
+        success ?
+            <div>Succesfully submitted</div>
+            :
+        <Form.Root
```
Comment (kerim1) [inline src/review.tsx:19]: Can you give this a `classname` attribute? Something like `className="reviewForm"`? That makes the styling easier for someone using this component.

**Review by kerim1 — CHANGES_REQUESTED** [review]
Just some small comments and then it is ready to merge.

File: src/review.tsx (hunk 5 above)
Comment (ekoi) [reply src/review.tsx:19]: Done

File: src/StarRating.tsx (hunk 1 above)
Comment (kerim1) [reply src/StarRating.tsx:14]: They are still there?

File: src/review.tsx (hunk 5 above)
Comment (kerim1) [reply src/review.tsx:19]: I meant the `Form.Root`

**Review by kerim1 — CHANGES_REQUESTED** [review]

File: src/review.tsx (hunk 3 above)
Comment (kerim1) [reply src/review.tsx]: Is still there?

**Pull request merged**